client/src/commands/run.ts

```diff
@@ -66,14 +68,25 @@ async function _run(selected = false) {
   );
 }
 
+async function _run(selected = false) {
```

Comment:
Thank you for making this update.
Looks like this function doesn't need to be async as it doesn't await anything.
Either remove async or await runCode seems fine.